daos/movies_test.go

```diff
@@ -0,0 +1,38 @@
+package daos
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/ashwinspg/explore-golang/dtos"
+	"github.com/ashwinspg/explore-golang/test"
+)
+
```

Comment:
imports we usually follow this order:
1. inbuilt packages
2.  current repo packages
3.  third party libraries packages

So, the import will be

```suggestion
	"testing"
	
	"github.com/ashwinspg/explore-golang/dtos"
	"github.com/ashwinspg/explore-golang/test"

	"github.com/stretchr/testify/suite"	
```

Reply:
I tried to follow the order:

1. inbuilt packages
2. third party libraries packages
3. current repo packages

I'll change as you suggested.